spec/features/create_new_user_requests_spec.rb

```diff
@@ -63,7 +63,7 @@
     fill_in "User's name", with: "Bob Fields"
     fill_in "User's email", with: "bob@gov.uk"
     select "Cabinet Office (CO)", from: "User's organisation"
-    fill_in "What additional publishing permissions does the user need?", with: "XXXX"
+    fill_in "List any other publishing applications and permissions the user needs. If you’re not sure what these are, explain what tasks they need to be able to do.", with: "XXXX"
```

Comment:
I wonder if this field could be selected by id, rather than label text, to remove the need for the same text to be present in two places?  The [`fill_in` method](https://rubydoc.info/github/jnicklas/capybara/Capybara/Node/Actions:fill_in) appears to support this.